### DISCUSSION TITLE
9월 둘째 주 스프린트[09/12-09-18]

### DISCUSSION THREAD

**Discussion by HyejinYang** [team-mokcha/corabal-beta#22] [스프린트 일지]
### [2021.09.13]

- 생략

#### @HyejinYang 👩‍🚀 

1. 작업 사항
    - [x] 0잔 기록 백엔드, 프론트엔드 기능 구현 완료
    - [x] 전역 상태 관리가 필요하다고 판단해 store에 globalCoffeeLogState 추가
    - [x] 메인에서의 에러 노출을 위해 react-toast-message 라이브러리 설치 및 적용

2. 논의 사항
    - [x] 브랜치 관리법 변경 이후 아래와 같이 devlop에 새 브랜치가 통합되는..? 듯이 보임..? 괜찮은지 논의할 필요가 있어 보임     
![image](https://user-images.githubusercontent.com/70951555/133256502-bf017f0b-3536-4a2e-a6df-6316bf95240b.png)
    - [x]  당일 커피 로그 수정은 수연님이 담당하신 부분이지만, 커피 로그 생성 기능 구현과 연관이 있어 논의 필요. 
            (현재 당일 커피 로그를 가져오고 수정하는 기능이 아직 구현되지 않은 상태인데, '오늘 마감' 기능 완성을 위해서는 미리 해당 기능이 완성될 필요가 있음)
[이슈 14번 - 커피 생성](https://github.com/team-mokcha/corabal-beta/issues/14)
[이슈 9번 - 커피 수정](https://github.com/team-mokcha/corabal-beta/issues/9)

### [2021.09.17] 스프린트

1. 이슈 #14 관련 얘기하기
    - globalCoffeeLogState 전역 관리 불필요할 수 있음. -> 인지했음. 전역으로 관리하는 상태는 굳이 많이 만들지 않기.
    - navigation("index") 에러 해결 -> stack navigator 에 Main 추가
2. 혜진님은 이제 프로젝트에 투입할 시간이 별로 없다고 하심

**Comment by ghost** [comment]
2-1. 브랜치가 디벨롭 브랜치로 통합되는 건 약간 문제가 있어보이는데 자세한 건 보고 얘기해야겠네요!
2-2. 그러면 이번주 스프린트 목표를 커피 수정으로 바꿀 수 있는데 문제는 금요일부터 작업 가능할 것 같아요 괜찮으신가요?

**Comment by ghost** [comment]
2-1. 
깃 그래프 테스트 했을 때 <develop 브랜치로 자동 병합되는 현상>이 발생하지 않음
본문의 혜진님이 겪은 이슈는 일반적으로 발생하는 상황은 아닌 것 같음 (추측)
![image](https://user-images.githubusercontent.com/73899253/133357177-33f080df-59d9-4720-ab8c-5c3aaa1f9424.png)
![image](https://user-images.githubusercontent.com/73899253/133357192-11cbafbc-7a91-4d63-94f9-624f72c88207.png)
![image](https://user-images.githubusercontent.com/73899253/133357202-e7e3a8c6-e392-43b2-a6a9-f440b030a29b.png)

**Comment by ghost** [comment]
2-2.
기능 스왑
캘린더 백엔드 로직 -> 수연 #15 
당일 커피 로그 수정 백엔드 로직 -> 혜진 #9

**Comment by ghost** [comment]
3. 컴포넌트 분리
   - `screens` 폴더에는 스크린만 남기기
   - 해당 screens 안에 너무 많은 내용이 들어가서 파일 분리해야 할 경우, `components` 폴더 안에서 분리하기

**Comment by ghost** [comment]
4. `services` 폴더 안의 네이밍 컨벤션: `xxxx-serviece.tsx`
